src/main/scala/org/zalando/spearheads/innkeeper/services/EskipRouteService.scala

```diff
@@ -0,0 +1,64 @@
+package org.zalando.spearheads.innkeeper.services
+
+import java.time.LocalDateTime
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.google.inject.Inject
+import org.zalando.spearheads.innkeeper.api.JsonProtocols._
+import org.zalando.spearheads.innkeeper.api.{EskipRoute, EskipRouteWrapper, NewRoute, RouteName}
+import org.zalando.spearheads.innkeeper.dao.{PathRow, RouteRow, RoutesRepo}
+import spray.json.pimpString
+
+class EskipRouteService @Inject()(routesRepo: RoutesRepo, routeToEskipTransformer: RouteToEskipTransformer) {
+
+  private val -> = "\n -> "
```

Comment:
I won't inline it, as there are many usages of it :) when somebody changes it in one place, it has to be changes everywhere else...